src/main/java/com/github/manolo8/darkbot/core/manager/FacadeManager.java

```diff
@@ -95,6 +98,10 @@ public long getProxyAddressOf(String key) {
         return proxies.getPtr(key);
     }
 
+    public long getMediatorAddressOf(String key) {
```

Comment:
no, we're not exposing mediators outside of this class, and i'd rather actually change the register methods to be private, this WILL break i do not want 3rd parties using it. If there is a good use-case (and there's PLENTY) then register the mediator on the bot directly and expose it as an api